murfi-rt-PyProject/subjects/template/xml/xml_orig/2vol.xml

```diff
@@ -1,6 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <study name="rtDMN">
-  <option name="subjectsDir"> ../subjects </option>
   <subject>
```

Comment:
I think you said you are sending the subject name in on the cli to murfi? If that's the case you can remove the `<subject>` tag also.

Reply:
Good point! We do set `MURFI_SUBJECTS_DIR` and `MURFI_SUBJECT_NAME` via the CLI, so I think we can